dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/uploadconfig/elements/UploadCellConfigBuilder.kt

```diff
@@ -11,7 +11,7 @@ import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkBoo
  */
 
 @Suppress("LongParameterList")
-class CellConfigBuilder(
+class UploadCellConfigBuilder(
```

Comment:
Why did you feel like renaming these classes and functions by giving them the "Upload" prefix?

Reply:
Because the "view-page"-generation also has a similar class which is also named `CellConfigBuilder`.
This makes it difficult to click through the code (sometimes you are confused at which one you are actually looking at).
In order to make it easier for code-readers to understand which CellConfigBuilder they are looking at, I decided to rename this one slightly different.

Reply:
I see. In the long run we should rename the view page versions similarly.

Reply:
To be closed.